observability/logging/logging-6.0/log6x-clf.adoc

```diff
@@ -5,3 +5,185 @@ include::_attributes/common-attributes.adoc[]
+    - name: <name>
+      type: parse
+    - openShiftLabels:
+        <key>: <value>
+      name: <name>
+      type: openShiftLabels
+    - drop:
+      - test:  
+      type: drop
+    - kubeAPIAudit:
+        omitResponseCodes:
+        omitStages:
+        rules:
+      name: <name>
+      type: kubeAPIAudit
+    - prune:
+        in:
+        notin:
+      name: <name>
+      type: prune
+  outputs:
+    - azureMonitor:
+        authentication:
+          sharedKey:
+        customerId: <spaceID>
+      name: <name>
+      type: azureMonitor
+    - cloudwatch:
+        authentication:
+          type: <awsAccessKey|iamRole>
+        groupName: {.log_type||"default"}
+        region: us-east-2
+      name: <name>
+      type: cloudwatch
+    - elasticsearch:
+        index: {.log_type|"default"}
+        url: <url>
+      name: <name>
+      type: elasticsearch
+    - http:
+        authentication:
+          username:
+          password:
+          token:
+        url: <url>
+      name: <name>
+      type: http
+    - kafka:
+        authentication:
+          sasl:
+        brokers:
+        topic:
+        url: <url>
+      name: <name>
+      type: kafka
+    - loki:
+        authentication:
+        labelKeys:
+        tenantKey:
+        url: <url>
+      name: <name>
+      type: loki
+    - lokiStack:
+        authentication:
+        target:
+      name: <name>
+      type: lokiStack
+    - googleCloudLogging:
+        authentication: 
+        id:
+        logId:
+      name: <name>
+      type: googleCloudLogging
+    - splunk:
+        authentication: 
+        index: '{.log_type|""}'
+        url: <url>
+      name: <name>
+      type: splunk
+    - syslog
+        rfc:
+        url: 
+      name: <name>
+      type: splunk
+    - otlp:
+        authentication: 
+        url:
+      name: <name>
+      type: splunk
+  pipelines:
+    - inputRefs:
+        - <input-name>
+        - application
+        - infrastructure
+        - audit
+      name: <pipeline-name>
+      filterRefs:
+        - <filter-name1>
+        - <filter-namex>
+      outputRefs:
+        - <output-name>
+  serviceAccount:
+    name: <account-name>
+----
+
+// All of these need to be validated by engineering for 6.0.
+include::modules/log6x-audit-log-filtering.adoc[leveloffset=+1]
+
+include::modules/log6x-content-filter-drop-records.adoc[leveloffset=+1]
+
+include::modules/log6x-content-filter-prune-records.adoc[leveloffset=+1]
+
+include::modules/log6x-input-spec-filter-audit-infrastructure.adoc[leveloffset=+1]
+
+include::modules/log6x-input-spec-filter-labels-expressions.adoc[leveloffset=+1]
+
+include::modules/log6x-input-spec-filter-namespace-container.adoc[leveloffset=+1]
+
+include::modules/log6x-multiline-except.adoc[leveloffset=+1]
+
+[id="log6x-CLF-samples_{context}"]
+== Use case samples
+// Should be an include, create: modules/log6x-use-metric-export.adoc - Associated JIRA?
+* Log File Metric Exporter as a Separate Deployment
```

Comment:
@libander Could you please provide the link that I can refer to add here as we discussed?

Reply:
https://github.com/openshift/cluster-logging-operator/blob/master/docs/administration/logfilemetricexporter.adoc

Reply:
As discussed with @libander , this will be covered as part of larger PR.